content/en/docs/contribute/new-content/contributing-content.md

```diff
@@ -0,0 +1,267 @@
+    - If you add `fixes #<000000>` or `closes #<000000>` to the description,
+    where `#<000000>` is the number of  an associated issue, GitHub
+    automatically closes the issue when the PR merges.
+    - Leave the **Allow edits from maintainers** checkbox selected.
+
+6.  Click **Create pull request**.
+
+## Large changes {#large-changes}
+
+If you're more experienced with Git, or if your changes are larger than a few lines,
+fork the repository first, clone it locally, and then open a pull request.
+
+### Fork the `kubernetes/website` repository
+
+The Kubernetes project only allows pull requests from a fork.
+
+1. Navigate to the [`kubernetes/website`](https://github.com/kubernetes/website/) repository.
+2. Click **Fork**.
+
+### Create a local clone and set the upstream
+
+To work from a fork, ensure you have [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed on your computer. You can also use a Git UI application.
+
+1. In a terminal window, use `git clone` to clone the your fork.
+
+    ```bash
+    git clone git@github.com/<github_username>/website
+    ```
+
+    The new directory `website` is created in your current directory, with
+    the contents of your GitHub repository. Your fork is your `origin`.
+
+2. Navigate to the new `website` directory. Set the `kubernetes/website` repository as the `upstream` remote.
+
+      ```bash
+      cd website
+
+      git remote add upstream https://github.com/kubernetes/website.git
+      ```
+
+4.  Confirm your `origin` and `upstream` repositories.
+
+    ```bash
+    git remote -v
+    ```
+
+    Output is similar to:
+
+    ```bash
+    origin	git@github.com:<github_username>/website.git (fetch)
+    origin	git@github.com:<github_username>/website.git (push)
+    upstream	https://github.com/kubernetes/website (fetch)
+    upstream	https://github.com/kubernetes/website (push)
+    ```
+
+There are three different copies of the repository when you work locally:
+`local`, `upstream`, and `origin`. Fetch both the `origin` and `upstream` remotes. This
+updates your cache of the remotes without actually changing any of the copies.
+
+  ```bash
+  git fetch origin
+  git fetch upstream
+  ```
+
+This workflow deviates from the one defined in the Community's [GitHub
+Workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md).
+In this workflow, you do not need to merge your local copy of `master` with `upstream/master` before
+pushing the updates to your fork. That step is not required in
+`kubernetes/website` because you are basing your branch on the upstream repository.
+
+You can now build the website locally For more information, see the `kubernetes/website` repository's
+[README.md](https://github.com/kubernetes/website/blob/master/README.md).
+
+### View your changes locally
+
+If you aren't ready to create a pull request but you want to see what your
```

Comment:
nit: line 155, Do you need,
`If you aren't ready to create a pull request`
instead,
To view you changes locally,